operator/test/test.sh

```diff
@@ -160,6 +160,17 @@ test_chains() {
     echo "[ERROR] Public key is not accessible" >&2
     exit 1
   fi
+
+  echo -n "  - Check Pod Security: "
+  sccs="$(kubectl get pod -o name -n tekton-chains | xargs -l -r kubectl get -o jsonpath='{.metadata.annotations.openshift\.io/scc}' -n tekton-chains)"
+  if [ "$sccs" = "restricted-v2restricted-v2restricted-v2" ]; then
```

Comment:
regexp expertise appreciated ...

I can cobble one together that matches any number of `restricted-v2` in  row, but it also allows a series of `restricted-v2` but then something else at the end i.e. `restricted-v2privilged`